### PR TITLE
A solution to fixed issue #10

### PR DESCRIPTION
**Pull request by XinyueZ**
However the fix is based on [PR](https://github.com/googlesamples/android-sunflower/pull/14).

### CODE REVIEW

File: app/src/main/java/com/google/samples/apps/sunflower/PlantListFragment.kt [hunk 1]
```diff
@@ -69,9 +86,20 @@ class PlantListFragment : Fragment() {
...
+        databinding: FragmentPlantListBinding,
+        adapter: PlantAdapter,
+        recyclerViewState: Parcelable?
+    ) {
         viewModel.getPlants().observe(this, Observer { plants ->
-            if (plants != null) adapter.values = plants
+            if (plants != null) {
+                adapter.values = plants
+                databinding.loadingUi.visibility = GONE
+
```
Comment (piotrek1543) [inline app/src/main/java/com/google/samples/apps/sunflower/PlantListFragment.kt]: If I were you, I would omit static import and change a line above to `databinding.loadingUi.visibility = View.GONE`

File: app/src/main/res/layout/fragment_plant_list.xml [hunk 2]
```diff
@@ -15,17 +15,33 @@
...
-        app:layoutManager="android.support.v7.widget.LinearLayoutManager"
-        tools:context="com.google.samples.apps.sunflower.GardenActivity"
-        tools:listitem="@layout/list_item_plant" />
+        android:paddingLeft="@dimen/margin_normal"
+        android:paddingRight="@dimen/margin_normal"
+        android:layout_height="match_parent">
+
+        <android.support.v7.widget.RecyclerView
+            android:id="@+id/plant_list"
+            android:name="com.google.samples.apps.sunflower.PlantListFragment"
```
Comment (piotrek1543) [inline app/src/main/res/layout/fragment_plant_list.xml]: Shouldn't `android:name` be added to parent, I mean to `FrameLayout` after your layout changes?

File: app/src/main/java/com/google/samples/apps/sunflower/PlantListFragment.kt (hunk 1 above)
Comment (XinyueZ) [reply app/src/main/java/com/google/samples/apps/sunflower/PlantListFragment.kt]: Ya. I will change.

File: app/src/main/res/layout/fragment_plant_list.xml (hunk 2 above)
Comment (XinyueZ) [reply app/src/main/res/layout/fragment_plant_list.xml]: Sorry you are right.

Comment (piotrek1543) [reply app/src/main/res/layout/fragment_plant_list.xml]: @XinyueZ I noticed also that in the second MR you have the same "issue"

**Comment by tiembo** [comment]
Closing this PR, as PR #33 addresses #10

**Pull request closed without merging**